src/EditorFeatures/Core.Wpf/Structure/VisualStudio14StructureTaggerProvider.cs

```diff
@@ -31,12 +30,13 @@ internal partial class VisualStudio14StructureTaggerProvider :
     {
         [ImportingConstructor]
         public VisualStudio14StructureTaggerProvider(
+            IThreadingContext threadingContext,
```

Comment:
No `[Obsolete]` attribute on this importing constructor?

Reply:
➡️ Same as previous; called by old test code.